scripts/install_vim.sh

```diff
@@ -1,88 +1,186 @@
 #!/bin/bash
 
 set -e
+set -x
```

Comment:
I don't feel strongly about it.  I think it can stay as is.  It just stuck out as something that might've been left behind from testing.